akka-projection-slick/src/main/scala/akka/projection/slick/internal/SlickOffsetStore.scala

```diff
@@ -111,7 +111,7 @@ import slick.jdbc.JdbcProfile
     if (useLowerCase) toRootLowerCase(str)
     else str
 
-  class OffsetStoreTable(tag: Tag) extends Table[OffsetRow](tag, slickSettings.schema, slickSettings.table) {
+  class OffsetStoreTable(tag: Tag) extends Table[OffsetRow](tag, slickSettings.schema, dialect.tableName) {
```

Comment:
Don't we need to uppercase the `slickSettings.schema` as well?

Reply:
I thought about that. But the default value is empty and if the user wants to set a custom value I hope that he knows what it does. 😄

Reply:
I see your point. 
I'm fine to leave it out. 

On the other hand, if they don't know what they are doing, they may add lowercase for both and be surprised that works for one and not for the other.

Reply:
Make sense. 🤔 
Ok, I will do that. 😉